circle.yml

```diff
@@ -1,8 +1,14 @@
 machine:
   services:
     - docker
+  environment:
+    ECR_ENABLED: True
 
 dependencies:
+  pre:
+    # get ECR creds
+    - pip install awscli==1.11.76
+    - $(aws ecr get-login --region $AWS_REGION)
```

Comment:
you need `--registry-ids` if you have credentials to multiple accounts. `get-login` will use the account id of the "home" of the credentials it has. Older repos had credentials in old-prod that had access to ecr in the ops account, so in that instance we need to specify the ops account id. But I updated everyones keys to ones that belong to ops, so this isn't actually needed anymore.